README.md

```diff
@@ -558,6 +558,7 @@ To develop locally, clone this repository and run `. script/bootstrap` to instal
     </tr>
     <tr>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/epenet"><img src="https://avatars.githubusercontent.com/u/6771947?v=4?s=100" width="100px;" alt="epenet"/><br /><sub><b>epenet</b></sub></a><br /><a href="https://github.com/syrupy-project/syrupy/commits?author=epenet" title="Tests">⚠️</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="http://about.me/tomsparrow"><img src="https://avatars.githubusercontent.com/u/793763?v=4?s=100" width="100px;" alt="Tom Sparrow"/><br /><sub><b>Tom Sparrow</b></sub></a><br /><a href="https://github.com/syrupy-project/syrupy/issues?q=author%3Asparrowt" title="Bug reports">🐛</a></td>
```

Comment:
It's not really important, just intrigued that "bug" means "bug report" rather than "bug fix"?